Description: fix shebang paths
 This fixes /usr/local paths in php shebangs...
Forwarded: no
Author: Richard Sellam <richard.sellam@orvidia.fr>
--- a/arcanist/src/lint/linter/__tests__/xhpast/php-tags-script.lint-test
+++ b/arcanist/src/lint/linter/__tests__/xhpast/php-tags-script.lint-test
@@ -1,4 +1,4 @@
-#!/usr/bin/local/php
+#!/usr/bin/php
 <?php
 
 return;
--- a/libphutil/support/xhpast/generate_nodes.php
+++ b/libphutil/support/xhpast/generate_nodes.php
@@ -1,4 +1,4 @@
-#!/usr/local/bin/php
+#!/usr/bin/php
 <?php
 
 $nodes = <<<EONODES
